generator/templates/view.lkml

```diff
@@ -0,0 +1,69 @@
+view: {{ view }} {
```

Comment:
On the one hand, this is a great use of Jinja.

On the other hand, looker supports [liquid variables](https://docs.looker.com/reference/liquid-variables), which is basically jinja templating in lookml. So soon we'll be supporting two layers of jinja templating. [It is possible](https://github.com/mozilla/bigquery-etl/blob/master/bigquery_etl/events_daily/query_templates/event_types_v1/metadata.yaml#L14), but it's not pretty and possible confusing.

No change required, just musing.

Reply:
yeah, i think i'm going to change this to serialize manually. for a bit there during development it was helpful, but now that I have `class Dimension`, manual serializing is seeming better.